src/SeqCli/Templates/Export/EntityName.cs

```diff
@@ -15,6 +15,11 @@ public static string FromEntityType(Type entityType)
 
     public static string ToResourceGroup(string resource)
     {
+        if (resource.Equals("expressionindex"))
+        {
+            return "expressionindexes";
+        }
```

Comment:
FWIW checking for trailing `x` and adding `es` would work in most cases, matching the `y` case below?